macosx/Torrent.mm

```diff
@@ -98,6 +98,14 @@ bool trashDataFile(char const* filename, void* /*user_data*/, tr_error* error)
 
 @implementation Torrent
 
++ (void)initialize
+{
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
```

Comment:
Based on the docs, `dispatch_once()` isn't necessary here. It's enough to check `self == [ClassName self]`.

Reply:
Well, we don't need to compare which one of the two solutions is better. Both are fine. `dispatch_once` conveys the best intent and is closer to modern programming (it's the equivalent of `static let`), so I prefer it.

Reply:
Anyway, unblocking users to get Transmission to work instead of crashing on startup is the priority, so here is the `self == [ClassName self]`: b1d72552dc6fb84aac67c0a7c4143b5a0b1f47d3